keyboards/meletrix/zoom_pad/info.json

```diff
@@ -0,0 +1,93 @@
+{
+    "keyboard_name": "zoom_pad",
+    "manufacturer": "meletrix",
+    "maintainer": "spbgzh",
+    "usb": {
+        "vid": "0x916B",
+        "pid": "0x0008",
+        "device_version": "0.0.1"
+    },
+    "features": {
+        "bootmagic": true,
+        "command": false,
+        "console": false,
+        "extrakey": true,
+        "mousekey": true,
+        "nkro": true,
+        "encoder": true
+      },
+    "matrix_pins": {
+        "cols": ["F1", "F4", "F0", "B5", "D0"],
+        "rows": ["D5", "F5", "F6", "F7", "C7", "B6"]
+    },
+    "diode_direction": "COL2ROW",
+    "encoder": {
+        "rotary": [
+            {"pin_a": "B4", "pin_b": "D3", "resolution": 2}
+        ]
+    },
+    "processor": "atmega32u4",
+    "bootloader": "atmel-dfu",
+    "layouts": {
+        "LAYOUT_all": {
```

Comment:
This should be reformatted to match the numpad_6x4 layout:
https://github.com/qmk/qmk_firmware/tree/master/layouts/default/numpad_6x4

And should be named as such.

https://docs.qmk.fm/#/feature_layouts